pkg/util/testing/wrappers.go

```diff
@@ -410,6 +410,13 @@ func MakePodSet(name kueue.PodSetReference, count int) *PodSetWrapper {
 							Resources: corev1.ResourceRequirements{
 								Requests: make(corev1.ResourceList),
 							},
+							SecurityContext: &corev1.SecurityContext{
```

Comment:
Please deduplicate by some function in util/testing, maybe `DefaultSecurifyContextForTest()`?

Reply:
Added a function for this.